nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/bulk/IndexCompletionComputation.java

```diff
@@ -93,9 +98,9 @@ protected void logIndexing(BulkStatus status) {
         long wait = status.getScrollStartTime().toEpochMilli() - status.getSubmitTime().toEpochMilli();
         long scroll = status.getScrollEndTime().toEpochMilli() - status.getScrollStartTime().toEpochMilli();
         double rate = 1000.0 * status.getTotal() / (elapsed);
-        log.warn(String.format(
-                "Index command: %s completed: %d docs in %.2fs (wait: %.2fs, scroll: %.2fs) rate: %.2f docs/s",
-                status.getId(), status.getTotal(), elapsed / 1000.0, wait / 1000.0, scroll / 1000.0, rate));
+        log.warn("Index command: {} completed: {} in {}", status.getId(), status.getTotal(),
+                String.format("%.2fs (wait: %.2fs, scroll: %.2fs) rate: %.2f docs/s", elapsed / 1000.0, wait / 1000.0,
```

Comment:
You can use `String.format` expression in log4j2 message. Code below should work:
```
log.warn("Index command: %s completed: %d in %.2fs (wait: %.2fs, scroll: %.2fs) rate: %.2f docs/s", 
    status.getId(), status.getTotal(), elapsed / 1000.0, wait / 1000.0, scroll / 1000.0, rate);
```

Reply:
Yep, t try it this morning when i did the modification unfortunately  it didn't work 
as example 
`LogManager.getLogger(DeprecationLoggerTest.class).warn("Index command: %s completed: %d in %.2fs (wait: %.2fs, scroll: %.2fs) rate: %.2f docs/s",
                1, 2, 3, 4, 10, 2);` 

will show `WARN  [DeprecationLoggerTest] Index command: %s completed: %d in %.2fs (wait: %.2fs, scroll: %.2fs) rate: %.2f docs/s`

for String.format use with log4j we can use `log.printf`

as example 
`LogManager.getLogger(DeprecationLoggerTest.class).printf(Level.WARN, "Index command: %s completed: %d in %.2fs (wait: %.2fs, scroll: %.2fs) rate: %.2f docs/s",
                1, 2, 3f, 4f, 10f, 2f);
    }`
and the result `[DeprecationLoggerTest] Index command: 1 completed: 2 in 3.00s (wait: 4.00s, scroll: 10.00s) rate: 2.00 docs/s`

to be consistent with the other code i chose to use `{}` instead of adding a new use case of printf